tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java

```diff
@@ -54,6 +61,9 @@
  */
 public class TransactionalMessageCopier {
 
+    private static final Logger log = LoggerFactory.getLogger(TransactionalMessageCopier.class);
```

Comment:
Do we really need to a a logger instead of just using `System.out.println()`? And if yes, should we update all outputs to use the logger? It's a little unclear why we would mix both approaches?

Reply:
Correct, this gets removed.